cache-tree.c

```diff
@@ -221,20 +221,31 @@ static void discard_unused_subtrees(struct cache_tree *it)
 	}
 }
 
-int cache_tree_fully_valid(struct cache_tree *it)
+static int cache_tree_fully_valid_recursive(struct cache_tree *it)
 {
 	int i;
 	if (!it)
 		return 0;
 	if (it->entry_count < 0 || !has_object_file(&it->oid))
 		return 0;
 	for (i = 0; i < it->subtree_nr; i++) {
-		if (!cache_tree_fully_valid(it->down[i]->cache_tree))
+		if (!cache_tree_fully_valid_recursive(it->down[i]->cache_tree))
```

Comment:
In Git's source code, we often use the suffix `_1` for things like that (see e.g. [`map_loose_object_1()`](https://github.com/git/git/blob/v2.22.0/sha1-file.c#L1017)).

If you plan on submitting this to the Git mailing list eventually, it might make sense to imitate that de facto convention.

Reply:
done